node.js

```diff
@@ -0,0 +1,9 @@
+// Conditional Exports are not yet widely supported.
```

Comment:
conditional exports are less of an issue than simply supporting named exports (which typescript does not)